src/test/java/bio/terra/service/auth/iam/ras/EcmServiceTest.java

```diff
@@ -19,38 +22,39 @@
 import java.nio.charset.StandardCharsets;
 import java.util.Base64;
 import org.hamcrest.Matchers;
-import org.junit.Before;
-import org.junit.Test;
-import org.junit.experimental.categories.Category;
-import org.junit.runner.RunWith;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
 import org.mockito.Mock;
-import org.mockito.junit.MockitoJUnitRunner;
+import org.mockito.junit.jupiter.MockitoExtension;
 import org.springframework.http.HttpStatus;
 import org.springframework.test.context.ActiveProfiles;
 import org.springframework.web.client.HttpClientErrorException;
 import org.springframework.web.client.RestTemplate;
 
 @ActiveProfiles({"google", "unittest"})
```

Comment:
I'm not grasping fully when we want `@ActiveProfiles` set and when we can remove them, can you help me understand?

Reply:
Yeah, my junit porting "toolbox" changed as I was going through all these files, so the changes aren't completely consistent. I can go back through and remove more `@ActiveProfiles` so the new code is consistent.

To answer your question (which I also wasn't sure of before working on this, but had been wondering for a while), this annotation only affects tests that use a spring context for injection. A spring profile can control which objects are injected, so it can be used to provide different behaviors in different situations, without needing to use named injections or other bean qualifiers. So if a test doesn't rely on `@Autowired` objects, then this annotation should have no effect on the test. Removing it makes that clearer and I wouldn't be surprised if there's some annotation processing overhead that we can avoid too.